pandapower/test/converter/test_from_cim.py

```diff
@@ -1,21 +1,71 @@
+# import os
```

Comment:
looks like this commented code can be removed

Reply:
Well,since this change is not so necessary , I prefer keep this "commented code" to keep up with the tutorial of "cim2pp.ipynb" , this code is more experimental rather than the original cods. and its's just tested with vscode in linux now, I didn't test it in NT system. By the way, I had tried to use the relative path, the "/" in posix and "\" in nt is frustrated, that is another reason I turn to the iterable function.  I'd like to comment here like that:" this fucntion is a optional method, the commented code above is more easy and simple for guys focus on prime concept. " , could this be done?